lib/karafka/routing/consumer_mapper.rb

```diff
@@ -17,7 +17,7 @@ module Routing
     #   module MyMapper
     #     def self.call(raw_consumer_group_name)
     #       [
-    #         Karafka::App.config.client_id.to_s.underscope,
```

Comment:
I guess we can't use ```underscore``` directly anymore right?

Reply:
Yeah. Unless you use Karafka together with Rails application or require ActiveSupport yourself.

Reply:
Ok, so we need to update this example as well.